Reflective Report.tex

```diff
@@ -0,0 +1,85 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\usepackage{graphicx}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{Reflective Essay}
+
+% Please do not change the subtitle
+\subtitle{COMP120 - Reflective Report}
+
+% Please put your student number in the author field
+\author{1503048}
+
+\begin{document}
+
+\maketitle
+
+\abstract{In this reflective report, I'll be reviewing my learning while reflecting on my projects and approaches for the first semester.}
+
+\section{Introduction}
+
+The three weakness I'll be exploring in this report are: time management, critical evaluation, . All of these skills are significant to me for professional practise and have been emphasised during my first semester.
+
+\section{Time Management}
+
+\subsection{Description - What happened? What is being examined?}
+
+One of my weaknesses was time management. With most tasks, it is crucial to follow a strict and thoughtful plan which accounts for all aspects of the project. For me, the relevance of this issue was for not anticipating all aspects of each project.
+
+\subsection{How has this impacted my learning and projects?}
+
+I found that my allocated work time had been misplaced. For me, most of the time was focused on research of ideas and techniques rather than practical applications. This not only lead to a low quality of work but also low quantity too. This uneven distribution also hindered my overall learning.
```

Comment:
Thinking and reflection are important tasks --- particularly with programming activities as most problem solving can *and is* done away from the keyboard. Is it, perhaps, the focus of this research and/or the way in which this research was recorded and utilized that lead to productivity challenges?